vendor/knative.dev/eventing/test/rekt/features/broker/feature.go

```diff
@@ -33,185 +32,16 @@ import (
 	"knative.dev/eventing/test/rekt/resources/channel"
 	"knative.dev/eventing/test/rekt/resources/subscription"
 	"knative.dev/eventing/test/rekt/resources/trigger"
-
-	v1 "knative.dev/pkg/apis/duck/v1"
+	eventingduckv1 "knative.dev/pkg/apis/duck/v1"
 	"knative.dev/pkg/ptr"
 
 	"knative.dev/reconciler-test/pkg/eventshub"
 	eventasssert "knative.dev/reconciler-test/pkg/eventshub/assert"
 	"knative.dev/reconciler-test/pkg/feature"
 	"knative.dev/reconciler-test/pkg/manifest"
-	"knative.dev/reconciler-test/pkg/resources/service"
+	"knative.dev/reconciler-test/resources/svc"
 )
 
-func BrokerWithManyTriggers() *feature.Feature {
```

Comment:
why the hell is this removed ?!

Reply:
@liuchangyan you added the func in https://github.com/knative/eventing/pull/6761

and we still use it, like:
https://github.com/knative-sandbox/eventing-kafka-broker/blob/main/test/e2e_new/broker_with_many_triggers_test.go#L44

What am I not seeing here?

Reply:
@creydr  Seems you are not add this function: https://github.com/knative-sandbox/eventing-kafka-broker/pull/2996

Reply:
> @creydr Seems you don't add this function: #2996

The function was added before already in https://github.com/knative-sandbox/eventing-kafka-broker/pull/2989